Multisig/UI/Settings/MFA/KeySecurityOverviewViewController.swift

```diff
@@ -0,0 +1,160 @@
+            case info(String, String)
+        }
+    }
+    
+    convenience init() {
+        self.init(namedClass: Self.superclass())
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        tableView.registerCell(SecurityFactorTableViewCell.self)
+        tableView.registerCell(WarningTableViewCell.self)
+        tableView.registerHeaderFooterView(BasicHeaderView.self)
+
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = 60
+
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.tableFooterView = UIView()
+        title = "Recovery Kit"
+
+        if #available(iOS 15.0, *) {
+            tableView.sectionHeaderTopPadding = 0
+        }
+
+        infoButton = UIBarButtonItem(image: UIImage(named: "ico-info-24"),
+                style: UIBarButtonItem.Style.plain,
+                target: self,
+                action: #selector(showHelpScreen))
+        navigationItem.rightBarButtonItem = infoButton
+    }
+
+    @objc func showHelpScreen() { 
+
+    }
+
+    override func reloadData() {
+        buildSections()
+        tableView.reloadData()
+    }
+
+    private func buildSections() {
+        sections = []
+
+        // TODO: Build sections properly
+        sections.append(SectionItems(section: .enabledFactors("ENABLED FACTORS"), items: [
+            Section.Factor.factor("Email address", "ann.fischer@gmail", "ico-eMail", true, true)]))
+        sections.append(SectionItems(section: .otherFactors("OTHER FACTORS"),
+                                     items: [
+                                        Section.Factor.factor("Security password", nil, "ico-password", false, false),
+                                        Section.Factor.factor("Trusted device", nil, "ico-mobile", false, false)]))
+        sections.append(SectionItems(section: .info,
+                                     items: [Section.Info.info("More factors are coming soon!", "ico-clock")]))
+
+        let header = TableHeaderView(frame: CGRect(x: 0, y: 0, width: 0, height: 120))
+        header.set("Protect your owner from unauthorised access and ensure easy recovery. We recommend to enable at least 2 recovery factors.", centered: true, linesCount: 3, backgroundColor: .backgroundPrimary)
+        
+        tableView.tableHeaderView = header
+    }
+
+    func numberOfSections(in tableView: UITableView) -> Int {
+        sections.count
+    }
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        sections[section].items.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch sections[indexPath.section].section {
+        case .enabledFactors(_), .otherFactors(_):
+            if case let Section.Factor.factor(name, value, image, isDefault, selected) = sections[indexPath.section].items[indexPath.row] {
```

Comment:
Perhaps, for better readability the `sections[indexPath.section].items[indexPath.row]` can be extracted to a variable?